pkg/webui/console/containers/lora-cloud-gls-form/index.js

```diff
@@ -124,6 +155,15 @@ const LoRaCloudGLSForm = () => {
     throw error
   }
 
+  const initialQuery = initialValues.data.query
+  const [queryType, setQueryType] = useState(initialQuery)
+
+  const initialMultiFrame = initialValues.data.multi_frame
+  const [multiFrame, setMultiFrame] = useState(initialMultiFrame)
```

Comment:
nit
```suggestion
  const [multiFrame, setMultiFrame] = useState(initialValues.data.multi_frame)
```

Since `initialMultiFrame` is not used in any other place within the component